inc/cachify.class.php

```diff
@@ -1776,14 +1776,14 @@ public static function options_page()
 							<fieldset>
 								<label for="cachify_without_ids">
 									<input type="text" name="cachify[without_ids]" id="cachify_without_ids" value="<?php echo esc_attr($options['without_ids']) ?>" />
-									Post/Pages-IDs
+									__('Post/Pages-IDs (separated by commas)', 'cachify' );
```

Comment:
Also here, I will buy a blank tp `esc_html__( '` and a filter function to sanitize the input string.